cozy-theme/base.html

```diff
@@ -26,6 +26,7 @@
   <link rel="stylesheet" href="{{ base_url }}/css/highlight.css">
   {%- for path in extra_css %}
   <link href="{{ path }}" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
```

Comment:
Do you really want to allow Google to track our users every time they read our doc? Loading fonts from Google has some privacy issues, so I removed this font from the default theme.